apps/docs/content/3-components/2-library/25-modal.mdx

```diff
@@ -9,6 +9,92 @@ status: in-development
 
 <ComponentStatusBlock componentId="modal" />
 
+import { modalProps } from "@/components/navigation/modal"
+
+<Modal triggerButton={modalProps.triggerButton}  >
+{modalProps.children}
+</Modal>
+
+<Tabs id="tab1">
+  <TabList>
+    <TabItem value="tab11" checked>HTML</TabItem>
+    <TabItem value="tab12">Macro</TabItem>
+    <TabItem value="tab13">React</TabItem>
+  </TabList>
+  <TabPanel value="tab11">
+  ```html
+<div data-module="gieds-modal">
```

Comment:
Remove the `data-*` from the HTML samples; they will be more readable unless needed for the interaction.

Reply:
We can remove it from here since it is plain html with no interaction. But in a real case scenario the `data-*` is needed for the js logic